### PR TITLE
Form validation Milestone 

### PR DESCRIPTION
**Pull request by brainconnect93**
I did this project by
1. Create a new branch (form validation) and pull my work from the main branch
2. modify the form in my contact section to suit the project requirement
3. typed some JavaScript code to make the form validation
4. commit my changes
5. push my work to the repo

### CODE REVIEW

File: js/valid.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+function emailCheckLowerCase(e) {
+  if (email.value !== email.value.toLowerCase()) {
+    errorText.classList.remove('hide');
+    e.preventDefault();
+  } else {
+    errorText.classList.add('hide');
+  }
+}
+form.addEventListener('change', emailCheckLowerCase);
```
Comment (danmainah) [inline js/valid.js]: - [x] Kindly make sure your error message is only triggered when the user clicks on the submit button, currently, it appears if any changes happen in the form input fields and the email is not lowercase.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @brainconnect93 and @massaillon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: js/valid.js (hunk 1 above)
Comment (brainconnect93) [reply js/valid.js]: rectified

File: index.html [hunk 2]
```diff
@@ -157,21 +157,24 @@ <h2>Contact me</h2>
...
-                <textarea id="message" 
+                
+                <textarea class="form-input" id="message" 
                 name="message" maxlength="500" required>Write your message here
                 </textarea>
 
+                <div class="error-text hide">
+                    <em>Failed, kindly enter your email in lower case</em>
+                 </div>
                 <button type="submit" class="btn">Get It Touch</button>
```
Comment (sja-thedude) [inline index.html:178]: - [ ] Great job in implementing the changes, the error message appears when clicking on submit button, however, it also appears when clicked on text area. As suggested previously, kindly make sure your error message is only triggered when the user clicks on the submit button.

https://user-images.githubusercontent.com/70269507/169642954-3e56148e-f4c7-4476-9e3c-d38ee8dafc3b.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @brainconnect93, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Great job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
- [x] It is highly suggested to fix your text field as your form is getting submitted without adding any text field.

https://user-images.githubusercontent.com/70269507/169643322-aaa6218b-8aa7-4cfc-9687-33f3646b30c2.mov




_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (brainconnect93) [reply index.html:178]: okay thanks 
have done has you requested

**Review by phelian23 — APPROVED** [review]
Hi @brainconnect93, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**